yapf/yapflib/yapf_api.py

```diff
@@ -51,6 +52,7 @@
 
 def FormatFile(filename,
                style_config=None,
+               style_opts=None,
```

Comment:
oops, my editor is too helpful, the commit strips intentional trailing whitespace from a bunch of tests... I'll push a fix for this after your feedback